src/test/java/org/folio/processing/mapping/mapper/writer/marc/MarcRecordModifierTest.java

```diff
@@ -1492,7 +1492,28 @@ public void shouldProtectDataFieldIfTheyAreDuplicatedAndNotAllFieldsAreProtected
     testMarcUpdating(incomingParsedContent, existingParsedContent, expectedParsedContent, mappingParameters, mappingProfile);
   }
 
+  @Test
+  public void protectedFieldShouldRevealEvenIfItNotExistsInExistingRecord() throws IOException {
```

Comment:
propose "shouldAddIncomingFieldIftNotExistsInExistingRecordAndMatchesProtectionSetting" name  to describe the test case in terms of change/update operation of existing record